edison/templates/login.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
```

Comment:
It may be better to extend the existing 'layout.html' instead of creating a completely different one.

Reply:
Ill look into that when I have the time, kinda busy studying for exams at the moment